assets/master/0100_service_account.yaml

```diff
@@ -2,4 +2,4 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: nfd-master
-  namespace: openshift-nfd
+#  namespace: openshift-nfd
```

Comment:
Why are these comments?

Reply:
I will remove it now, the tests are green now.